src/components/Feedback/Feedback.tsx

```diff
@@ -51,8 +51,8 @@ const Feedback = () => {
 
       <Box
         as="textarea"
+        // @ts-expect-error -- Generic Box implementation isn't clever enough to accept React.MutableRefObject<HTMLTextAreaElement> here.
```

Comment:
`Box` won't adapt to accept refs for the various components that are passed above using `as=...`. We can live with this as we are migrating over time to use our design system anyway